base_classes/NXobject.nxdl.xml

```diff
@@ -30,5 +30,182 @@
+						The last character (a digit or 'X') serves as a check character, making the ISSN uniquely defined by its first seven digits.
+						
+						Syntax: NNNN-NNNC, where N a decimal digit character (i.e., in the set {0,1,2,...,9}), and C is in {0,1,2,...,9,X}
+						Example: 1234-5678 or 1234-567X
+					</doc>
+				</item>
+				<item value="ISSN-L">
+					<doc>
+						Linking ISSN
+						
+						The linking The ISSN, or ISSN-L, is a specific ISSN that groups the different media of the same serial publication.
+						
+						Syntax: NNNN-NNNC, where N a decimal digit character (i.e., in the set {0,1,2,...,9}), and C is in {0,1,2,...,9,X}
+						Example: 1234-5678 or 1234-567X
+					</doc>
+				</item>
+				<item value="ORCID">
+					<doc>
+						Open Researcher and Contributor identifier.
+						
+						ORCID provides a free and persistent identifier that uniquely distinguishes authors and contributors in scientific research.
+						
+						Example: https://orcid.org/XXXX-XXXX-XXXX-XXXX
+						Syntax: https://orcid.org/0000-0002-1825-0097
+					</doc>
+				</item>
+				<item value="PURL">
+					<doc>
+						Persistent Uniform Resource Locator.
+						
+						A Persistent Uniform Resource Locator (PURL) is a type of URL designed to provide a stable, long-term reference to a web 
+						resource by using a resolver to redirect users to the resource's current location, even if it moves over time.
+						
+						A PURL has three parts: (1) a protocol, (2) a resolver address, and (3) a name.
+						
+						Syntax: https://purl.org/foo/bar
+						Example: https://purl.org/dc/elements/1.1/title
+					</doc>
+				</item>
+				<item value="ROR">
+					<doc>
+						Research Organization Registry
+						
+						A ROR ID is a globally unique identifier for research organizations, enabling unambiguous linking of institutions across systems.
+						
+						Syntax: https://ror.org/{ROR-ID}
+						Example: https://ror.org/052gg0110
+					</doc>
+				</item>
+				<item value="URL">
+					<doc>
+						Uniform Resource Locator
+						
+						Also known as web address, a URL is the address used to access a resource on the internet, 
+						specifying its location and the protocol to retrieve it.
+						
+						Syntax: scheme://domain:port/path?query_string#fragment_id
+						Example: https://www.example.com/about
+					</doc>
+				</item>
+				<item value="URN">
+					<doc>
+						Uniform Resource Name
+						
+						A URN is a unique, persistent identifier for a resource regardless of where it is stored.
+
+						It is recommended that identifiers with more specific type attribute (such as DOI or ISSN) values should not be stored as a URN,
+						even when this is valid. As an example, the URN doi:10.1107/S1600576714027575 is a valid URN-based representation for the DOI
+						10.1107/S1600576714027575, but it is strongly recommended to use type="DOI" in this case.
+						
```

Comment:
"strongly recommended" : is "strongly" needed?

Reply:
Not quite my choice of words but the goal when this was written was to emphasize this recommendation.  A wordsmith is most welcome to propose an improvement.

Reply:
I agree, "strongly" is probably a bit too much. I just changed it to "recommended".